places-rest-apis/adobe-i-o-integration/README.md

```diff
@@ -91,7 +102,7 @@ To create a Places integration:
 
 #### Generate a JWT token
 
-On the **JWT** tab, the Adobe I/O console allows you to test your integration by generating a JWT and providing the exchange URL.
+On the integration details page, click the **JWT** tab, the Adobe I/O console allows you to test your integration by generating a JWT and providing the exchange URL.
```

Comment:
Please restructure as follows:
"On the integration details page, on the **JWT** tab. the Adobe I/O console allows you to test your integration by generating a JWT and providing the exchange URL.